website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -146,6 +147,8 @@ The important entries to update for the manifest are:
 
 - `dataSources.source.startBlock`: the optional number of the block that the data source starts indexing from. In most cases, we suggest using the block in which the contract was created.
 
+- `dataSources.source.endBlock`: the optional number of the block that the data source stops indexing at. When provided the dataSource stops indexing at the specified block.
```

Comment:
is the block inclusive or exclusive? (i.e. in the example case does it index block 7175245 and stop, or does it stop before indexing 7175245?)

Reply:
Additionally I think worth specifying the `specVersion` from which this is available